### PR TITLE
chore: move over the helm chart for the ui from the operator repo

### PR DESCRIPTION
**Pull request by HarikrishnanBalagopal**
Signed-off-by: Harikrishnan Balagopal <harikrishmenon@gmail.com>

### CODE REVIEW

**Review by ashokponkumar — CHANGES_REQUESTED** [review]
Won't we need a copy of the helm chart here retained till the new operator is in place? Otherwise the current helm based operator is not valid. So we can only remove the GitHub action part alone. Also from operator perspective, in read me, the helm chart reference can possible be removed.

**Comment by ashokponkumar** [comment]
Also the prepare for release section from mahespur might need to be removed

**Comment by HarikrishnanBalagopal** [comment]
> Also the prepare for release section from mahespur might need to be removed

done

**Comment by HarikrishnanBalagopal** [comment]
> Won't we need a copy of the helm chart here retained till the new operator is in place? Otherwise the current helm based operator is not valid. So we can only remove the GitHub action part alone. Also from operator perspective, in read me, the helm chart reference can possible be removed.

aren't the release tar files enough?

**Comment by ashokponkumar** [comment]
The operator Dockerfile uses the helm chart from its context (the operator repo).

**Comment by HarikrishnanBalagopal** [comment]
> The operator Dockerfile uses the helm chart from its context (the operator repo).

I commented that out https://github.com/konveyor/move2kube-operator/pull/55/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R6

**Comment by ashokponkumar** [comment]
> > The operator Dockerfile uses the helm chart from its context (the operator repo).
> 
> I commented that out https://github.com/konveyor/move2kube-operator/pull/55/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R6

Yes, that line is required. Without it, the operator does nothing.

**Comment by HarikrishnanBalagopal** [comment]
> > > The operator Dockerfile uses the helm chart from its context (the operator repo).
> > 
> > 
> > I commented that out https://github.com/konveyor/move2kube-operator/pull/55/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R6
> 
> Yes, that line is required. Without it, the operator does nothing.

yes, but isn't it broken anyway?

**Comment by ashokponkumar** [comment]
We have not tested it extensively with the new UI. But technically, it should be working, since the operator does nothing but the equivalent of a helm install

File: Makefile [hunk 1]
```diff
@@ -131,12 +131,6 @@ bundle: kustomize
...
 
-.PHONY: prepare-for-release
-prepare-for-release:
-	mv helm-charts/move2kube/Chart.yaml old
-	cat old | sed -E s/version:\ v0.1.0-unreleased/version:\ ${IMAGE_TAG}/ | sed -E s/appVersion:\ latest/appVersion:\ ${IMAGE_TAG}/ > helm-charts/move2kube/Chart.yaml
-	rm old
-
 .PHONY: helm-install
 helm-install:
 	helm upgrade --install --set image_tag=${IMAGE_TAG} ${RELEASE_NAME} helm-charts/move2kube/
```
Comment (ashokponkumar) [inline Makefile:136]: Do we want to remove this section too?

**Review by ashokponkumar — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (HarikrishnanBalagopal) [reply Makefile:136]: let's retain it

**Review by ashokponkumar — APPROVED** [review]

**Pull request merged**